### ISSUE TITLE
destroy doesn't remove style: min-height from parent div

### ISSUE THREAD

**Issue by az143** [apexcharts/apexcharts.js#1654]
version 3.19.2:
after calling destroy on a chart the parent div is empty as expected, but still has
an inline style="min-height: <number> px;", and thus still occupies space on the page.

the desired outcome would be that destroy gets rid of all traces of the chart. (example use case: multiple synced charts where only some should be shown b/c of size constraints)

**Comment by junedchhipa** [comment]
Fixed

**Comment by Knogobert** [comment]
Thanks for fixing the initial bug, but this has unfortunately introduced another one for us. Going from `3.19.2` to `3.19.3`.
Just as [vue-apexcharts ISSUE#256](https://github.com/apexcharts/vue-apexcharts/issues/256) writes it.

I think perhaps a check like this in `Destroy.js:50` (`clearDomElements()`) could solve it:
```
if (this.w.globals.dom.Paper.node.parentNode && this.w.globals.dom.Paper.node.parentNode.parentNode) {
  this.w.globals.dom.Paper.node.parentNode.parentNode.style.minHeight =
      'unset'
}
```

Or even tidier with optional chaining:
```
if (this.w.globals.dom.Paper.node.parentNode?.parentNode) {
  this.w.globals.dom.Paper.node.parentNode.parentNode.style.minHeight =
      'unset'
}
```

**Comment by junedchhipa** [comment]
@Knogobert Thanks for reporting, fixed it!

**Comment by PhilippMeissner** [comment]
@junedchhipa When can we expect a new release with the issue reported by @Knogobert fixed? :)

**Comment by junedchhipa** [comment]
@PhilippMeissner In 3-5 days

**Comment by PhilippMeissner** [comment]
I don't want to stress, but any news @junedchhipa ? 😄

**Comment by junedchhipa** [comment]
Released v3.20.0
